pkg/boot/universalpayload/handoffblock.go

```diff
@@ -0,0 +1,235 @@
+// Copyright 2024 the u-root Authors. All rights reserved
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+package universalpayload
+
+import (
+	"strings"
+	"unsafe"
+
+	guid "github.com/google/uuid"
+	"github.com/u-root/u-root/pkg/boot/kexec"
+)
+
+// Efi HOB Handoff Info Table
+const EfiHobHandoffTableVersion = 0x0009
+const EfiHobAlignment = 0x100000
+
+// Efi HOB Types
+const (
```

Comment:
EFI HOB maybe?

Reply:
These are shortened forms, should we change them to EFI and HOB in comments and constants variables like this:

```go
// EFI HOB Types
const (
    EFIHOBTypeHandoff  = 0x001
)
```

Reply:
It'd also be good if they had a Type.

```go
type EFIHOBType uint64 // or whatever the right one is

const (
  EFIHOBTypeHandoff EFIHOBType = 0x0001
)
```

etc

Reply:
Good advice, now we can follow this code style.

Reply:
These style changes affect multiple commits, it's hard to split and rebase them into previous commits correspondingly. So I made a new commit for style changes. Is that ok?